dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSHPCCFile.java

```diff
@@ -192,6 +192,43 @@ public final void testGetProjectedRecordDefinition() throws HpccFileException
         Assert.assertEquals(3, projectedRecordDefinition.getNumDefs());
     }
 
+    private boolean testFieldDefsNotEquals(FieldDef fieldDef1, FieldDef fieldDef2)
```

Comment:
I found the logic polarity of this function a bit confusing. Could this be renamed (and restructured) as "areFieldDefsEqual(...)"

Reply:
@rpastrana Agreed, I renamed it to fieldDefinitionsAreSeparate, after thinking about it I wanted to avoid saying equal here because I am really checking if the objects are not the same.